src/FsCodec.SystemTextJson/StringConverter.fs

```diff
@@ -0,0 +1,18 @@
+namespace FsCodec.SystemTextJson
+
+open System.Text.Json.Serialization
+
+type StringConverter() =
+    inherit JsonConverter<string>()
+
+    override _.HandleNull = true
+    override _.CanConvert(t) = t = typeof<string>
+
+    override this.Read(reader, _typeToConvert, _options) =
+        let value = reader.GetString()
+        if value = null then failwith "Expected string, got null."
```

Comment:
message is fine, but probably invalidOp, nullArg, ArgumentOutOfRangeException or someting (not sure on prior art for a good exception type for this in the context of a JSON lib, maybe FSharp.STJ has picked an approach?)